Week_04/Lecture_8/assignment3/src/main/java/org/assignment1/assignment3/repository/primaryEmployeeRepository.java

```diff
@@ -0,0 +1,44 @@
+package org.assignment1.assignment3.repository;
+
+import org.assignment1.assignment3.model.employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class primaryEmployeeRepository {
```

Comment:
Why class name is starting by lower character? Pls review all the other classes.

Reply:
I updated all the class name with CamelCase convention from lecture 8 Assignment 1, 2, and 3, thank you Ms for the feedback.